admin-dev/themes/new-theme/js/pages/product/components/categories/tags-renderer.ts

```
@@ -31,7 +31,7 @@ const ProductCategoryMap = ProductMap.categories;
 export default class TagsRenderer {
   eventEmitter: EventEmitter;
 
-  container: HTMLElement;
+  container: HTMLElement | null;
```

Can't we set it as optional instead?

---

Same here, If I am correct (I might not remember well) shouldn't this be mandatory to exist? If there is no container, then the component won't work anyway, so why making it nullable or optional and hiding the actual error everywhere with the `container?.querySelector...`.? I think we should be more explicit in js code as we do in php (if its possible). And instead of hiding errors, show them as fast as possible, no?